src/events.ts

```diff
@@ -53,7 +62,44 @@ type DreamsEvent = IdTokenDidExpireEvent |
                    ExitRequestedEvent |
                    ShareEvent;
 
+type AccountProvisionInitiatedEvent = {
+  event: 'accountProvisioned';
```

Comment:
Shouldn't it be `messages.accountProvisionInitiated`? If so shouldn't `messages` be renamed to `event`?

Reply:
good point, renamed it :)